modules/account-lib/package.json

```diff
@@ -42,6 +42,7 @@
     "@bitgo/bls": "^0.1.0",
     "@bitgo/bls-dkg": "^1.0.2",
     "@bitgo/statics": "^6.17.0",
+    "@bitgo/sdk-coin-cspr": "^1.0.0",
```

Comment:
make sure to remove all the dependencies that are no longer required by this package. `casper-js-sdk` and possibly others

Reply:
I removed it from account-lib dependencies.